hatch_pip_compile/plugin.py

```diff
@@ -347,3 +346,12 @@ def pipools_environment_dict(self) -> Dict[str, Any]:
         Get the environment dictionary
         """
         return self.metadata.hatch.config.get("envs", {})
+
+    def plugin_check_command(self, *args, **kwargs) -> subprocess.CompletedProcess:
+        """
+        Run a command from the virtualenv
+        """
+        return self.virtual_env.platform.check_command(
+            *args,
+            **kwargs,
+        )
```

Comment:
Separating this into its own function made this manageable to patch in testing - I'm trying to decide whether to generate a new release for this change since it's the only `hatch-pip-compile` code change in the PR